decidim-admin/app/assets/javascripts/decidim/admin/sort_steps.js.es6

```diff
@@ -6,6 +6,13 @@
 // should have a `data-sort-url` attribute with the URL where the data should
 // be posted to.
 $(() => {
+ sortSteps();
+});
+$( document ).on('turbolinks:load' , function() {
+  sortSteps();
+});
+
+function sortSteps(){
   const sortableElement = $('#steps tbody');
 
   if (sortableElement[0]) {
```

Comment:
Oh, and I forgot I recommend naming your jQuery selectors like this: `const $sortableElement = ...` so you now that variable stores a jQuery object.